### ISSUE TITLE
NTR: beta-hydroxybutyrate (ketones) low in blood inappropriately (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8320]
New term request beta-hydroxybutyrate (ketones) low in blood inappropriately (elthat_130306175733):

**Comment by pnrobinson** [comment]
@MickeySegal
see #8319

**Comment by MickeySegal** [comment]
Your assumption in https://github.com/obophenotype/human-phenotype-ontology/issues/8321 is correct.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000236
- label: Inappropriately low concentration of beta-hydroxybutyrate
- parents: [Abnormal homeostasis (HP:0012337)](https://hpo.jax.org/app/browse/term/HP:0012337)
- pmid: 
- def: Lower than appropriate concentration of beta-hydroxybutyrate in states such as hypoglycemia in which the normal response would be to elevate acetoacetate.
- comment: Ketone synthesis in the liver produces acetoacetate and beta-hydroxybutyrate from two acetyl CoA molecules. This occurs during ketoacidosis in physiological (starvation) and disease-associated (diabetic ketoacidosis) states.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8320
- synonyms:
